src/EditorFeatures/Core.Wpf/InlineHints/InlineHintsTag.cs

```diff
@@ -156,15 +162,11 @@ public async Task<IReadOnlyCollection<object>> CreateDescriptionAsync(Cancellati
                 Background = format.BackgroundBrush,
                 Child = block,
                 CornerRadius = new CornerRadius(2),
-                Height = textView.LineHeight - (0.25 * textView.LineHeight),
-                HorizontalAlignment = HorizontalAlignment.Center,
-                Margin = new Thickness(left, top: -0.20 * textView.LineHeight, right, bottom: 0),
-                Padding = new Thickness(1),
-
-                // Need to set SnapsToDevicePixels and UseLayoutRounding to avoid unnecessary reformatting
-                SnapsToDevicePixels = textView.VisualElement.SnapsToDevicePixels,
-                UseLayoutRounding = textView.VisualElement.UseLayoutRounding,
```

Comment:
❔ Is this no longer the case?

Reply:
nope.  and it's actually problematic. the highlight rect doesn't do this, and it means we more often than not show up misaligned with it.